tasklist/importer-870/src/main/java/io/camunda/tasklist/zeebeimport/v870/processors/es/ProcessInstanceZeebeRecordProcessorElasticSearch.java

```diff
@@ -72,22 +65,16 @@ public class ProcessInstanceZeebeRecordProcessorElasticSearch {
   @Qualifier("tasklistFlowNodeInstanceTemplate")
   private FlowNodeInstanceTemplate flowNodeInstanceIndex;
 
-  @Autowired private TasklistListViewTemplate tasklistListViewTemplate;
-
   public void processProcessInstanceRecord(final Record record, final BulkRequest bulkRequest)
       throws PersistenceException {
 
     final ProcessInstanceRecordValueImpl recordValue =
         (ProcessInstanceRecordValueImpl) record.getValue();
-    if (isVariableScopeType(recordValue) && FLOW_NODE_STATES.contains(record.getIntent().name())) {
+    if (environment.isTestProfileEnabled()
```

Comment:
Do we still need the `test` profile flag here and on the other processors as well? If the CamundaExporter is enabled, these processors will not be interfering :thinking:

Reply:
Yes, if the CamundaExporter is enabled, then this would not be required anymore; we event could then delete those processors. I can check in the next PR (the last entity to move), if I can remove this flag now or not.